src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -85,10 +88,10 @@ status_check() {
     check_dispatch $1 beta rls src/tools/rls
     check_dispatch $1 beta rustfmt src/tools/rustfmt
     check_dispatch $1 beta clippy-driver src/tools/clippy
-    # these tools are not required for beta to successfully branch
+    # These tools are not required on the beta/stable branches.
+    # They will still cause failure during the beta cutoff week, see `src/tools/publish_toolstate.py` for that.
```

Comment:
```suggestion
    # They will still cause failure during the beta cutoff week, see `checkregression.py` for that.
```
I realized this 1minute too late. ;)